spec/views/comments/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "comments/edit", type: :view do
+  before(:each) do
+    @comment = assign(:comment, Comment.create!(
+      :body => "MyString"
+    ))
+  end
+
+  it "renders the edit comment form" do
+    render
+
+    assert_select "form[action=?][method=?]", post_comment_path(@comment), "post" do
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)